example_with_tags_lang_test.go

```diff
@@ -14,6 +14,7 @@ func Example_withTagsLang() {
 		StringCHI string `faker:"lang=chi"`
 		StringRUS string `faker:"lang=rus"`
 		StringJPN string `faker:"lang=jpn"`
+		StringEMJ string `faker:"lang=emj"`
```

Comment:
Hey @geshtng thanks for the PR.

I'm curious, is there any ISO code for this language?
I tried to find one, but couldn't find one. 🤔

Reply:
Actually there is none for `EMJ` (reference: [ref-1](https://www.loc.gov/standards/iso639-2/php/code_list.php), [ref-2](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))
I want to make new tag like `emot`, but it's a bit useless (I think that "I just want a random emote, not a specific emote").
That's why I'm still using `lang` tag.